doc/user/content/sql/alter-cluster.md

```diff
@@ -6,44 +6,159 @@ menu:
+      SCHEDULE = { MANUAL | ON REFRESH (...) }
+  )
+  [WITH ({ WAIT UNTIL READY({TIMEOUT | ON TIMEOUT {COMMIT|ROLLBACK}}) | WAIT FOR <duration> })]
+  ;
+  ```
 
-### Cluster options
+You must have ownership of the cluster. See also [Required
+privileges](#required-privileges).
 
-{{% cluster-options %}}
+{{< /tab >}}
+{{< tab "Reset to default" >}}
 
-### `with_options`
+To reset a cluster configuration back to its default value:
 
-{{< diagram "alter-cluster-set-with-options.svg" >}}
+  ```mzsql
+  ALTER CLUSTER <cluster_name>
+  RESET (
+      REPLICATION FACTOR,
+      INTROSPECTION INTERVAL,
+      INTROSPECTION DEBUGGING,
+      MANAGED,
+      SCHEDULE
+  )
+  ;
+  ```
 
-Field                         | Value                 | Description
-------------------------------|-----------------------|-------------------------------------
-**WAIT UNTIL READY...TIMEOUT**                   | `duration`            | For a [graceful cluster resizing](#graceful-cluster-resizing), the maximum duration to wait for the new replicas to be ready.
-**WAIT UNTIL READY...ON TIMEOUT**                | [`COMMIT`,`ROLLBACK`] | For a [graceful cluster resizing](#graceful-cluster-resizing), the action to take on timeout. <br><ul><li>`COMMIT` cuts over to the new replica regardless of its hydration status, which may lead to downtime.</li><li>`ROLLBACK` removes the pending replica and returns a timeout error.</li></ul>Default: `COMMIT`.
-**WAIT FOR**                  | `duration`            | For a [graceful cluster resizing](#graceful-cluster-resizing), a fixed duration to wait for the new replicas to be ready. This option might lead to downtime, so we recommend using the `WAIT UNTIL READY` option instead.
+You must have ownership of the cluster. See also [Required
+privileges](#required-privileges).
 
-## Graceful cluster resizing
+{{< /tab >}}
+{{< tab "Rename to" >}}
+
+To rename a cluster:
+
+  ```mzsql
+  ALTER CLUSTER <cluster_name> RENAME TO <new_cluster_name>;
+  ```
+
+You must have ownership of the cluster. See also [Required
+privileges](#required-privileges).
+
+{{< /tab >}}
+{{< tab "Alter owner to" >}}
+
+To change the owner of a cluster:
+
+  ```mzsql
+  ALTER CLUSTER <cluster_name> OWNER TO <new_owner_role>;
+  ```
+
+To rename a cluster, you must have ownership of the cluster and membership in
+the `<new_owner_role>`. See also [Required privileges](#required-privileges).
+
+{{< /tab >}}
+{{< tab "Swap names with another cluster" >}}
+
+To swap the name of this cluster with another cluster:
+
+  ```mzsql
+  ALTER CLUSTER <cluster1> SWAP WITH <cluster2>;
+  ```
+
+To swap names with another cluster, you must have ownership of both clusters.
+See also [Required privileges](#required-privileges).
+
+{{< /tab >}}
+{{< /tabs >}}
+
+### Cluster configuration
+
+{{% alter-cluster/alter-cluster-options %}}
+
+### `WITH` options
+
+{{< note >}}
+*Private preview* for [graceful cluster resizing](#graceful-cluster-resizing).
+{{< /note >}}
```

Comment:
```suggestion
```

Reply:
Common practice is to include the PrPr tag in the description of the configuration options.